geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureBlobStoreConfigProvider.java

```diff
@@ -60,36 +41,19 @@ public Object fromString(String str) {
             new StringConverter() {
                 @Override
                 public Object fromString(String str) {
-                    str = resolveFromEnv(str);
                     if (Strings.isNullOrEmpty(str)) {
                         return null;
                     }
                     return str;
                 }
             };
 
-    private static String resolveFromEnv(String str) {
-        if (gwcEnvironment == null) {
-            gwcEnvironment = GeoWebCacheExtensions.bean(GeoWebCacheEnvironment.class);
-        }
-        if (gwcEnvironment != null
-                && str != null
-                && GeoWebCacheEnvironment.ALLOW_ENV_PARAMETRIZATION) {
-            Object result = gwcEnvironment.resolveValue(str);
-            if (result == null) {
-                return null;
-            }
-            return result.toString();
-        }
-        return str;
-    }
-
     @Override
     public XStream getConfiguredXStream(XStream xs) {
         Class<AzureBlobStoreInfo> clazz = AzureBlobStoreInfo.class;
         xs.alias("AzureBlobStore", clazz);
-        xs.registerLocalConverter(clazz, "maxConnections", EnvironmentNullableIntConverter);
-        xs.registerLocalConverter(clazz, "proxyPort", EnvironmentNullableIntConverter);
+        xs.registerLocalConverter(clazz, "maxConnections", EnvironmentStringConverter);
```

Comment:
No need to register custom converters any longer, the default behavior should do (please check).

Reply:
Indeed not needed anymore, removed.